src/Microsoft.VisualStudio.ProjectSystem.Managed/ProjectSystem/DesignTimeTargets/Microsoft.Managed.DesignTime.targets

```diff
@@ -171,6 +171,10 @@
       <Context>File</Context>
     </PropertyPageSchema>
     
+    <PropertyPageSchema Include="$(ManagedXamlResourceDirectory)XamlAppDef.xaml">
```

Comment:
This property is missing an "s"; should be `$(ManagedXamlResourcesDirectory)`.

Reply:
Fixed, but it still doesn't seem to work. Please keep looking. Thanks!